### PR TITLE
Vt 1863 mpc changes

### PR DESCRIPTION
**Pull request by kunal-plivo**

### CODE REVIEW

File: plivo/xml/MultiPartyCallElement.py [hunk 1]
```diff
@@ -0,0 +1,548 @@
...
+    @property
+    def max_duration(self):
+        return self.__max_duration
+
+    @max_duration.setter
+    @validate_args(
+        max_duration=[
+            optional(
+                of_type_exact(*six.integer_types),
+                check(lambda max_duration: 300 <= max_duration <= 28800, '300 < max_duration <= 28800')
```
Comment (bhuvanvenkat-plivo) [inline plivo/xml/MultiPartyCallElement.py:18]: correct the failure message to `'300 <= max_duration <= 28800'`

File: plivo/xml/MultiPartyCallElement.py [hunk 2]
```diff
@@ -0,0 +1,548 @@
...
+    @property
+    def max_participants(self):
+        return self.__max_participants
+
+    @max_participants.setter
+    @validate_args(
+        max_participants=[
+            optional(
+                of_type_exact(*six.integer_types),
+                check(lambda max_participants: 2 <= max_participants <= 10, '2 < max_participants <= 10')
```
Comment (bhuvanvenkat-plivo) [inline plivo/xml/MultiPartyCallElement.py:37]: correct the failure message to `'2 <= max_participants <= 10'`

File: plivo/utils/validators.py [hunk 3]
```diff
@@ -62,16 +62,39 @@ def f(name, value):
...
+def is_in(iterable, message=None, case_sensitive=True, case_type='upper'):
     def f(name, value):
         actual_value = value
         if not case_sensitive:
-            value = str(value).upper()
+            if case_type == 'upper':
+                value = str(value).upper()
+            elif case_type == 'lower':
+                value = str(value).lower()
+            elif case_type == 'first_capital':
```
[the remark this answers is not in the document]
Comment (kunal-plivo) [reply plivo/utils/validators.py:73]: using title method here

**Comment by kunal-plivo** [comment]
@bhuvanvenkat-plivo taken care of review comments, please take a look

**Review by manish-plivo — APPROVED** [review]

**Pull request merged**